recipe-server/client/control/tests/state/revisions/test_reducers.js

```diff
@@ -0,0 +1,28 @@
+import { fromJS } from 'immutable';
+
+import {
+  REVISION_RECEIVE,
+} from 'control/state/action-types';
+import revisionsReducer from 'control/state/revisions/reducers';
+
+import {
+  INITIAL_STATE,
+  REVISION,
+} from '.';
+
+
+describe('Revisions reducer', () => {
+  it('should return initial state by default', () => {
+    expect(revisionsReducer(undefined, {})).toEqual(INITIAL_STATE);
+  });
+
+  it('should handle REVISION_RECEIVE', () => {
```

Comment:
The test descriptions for reducers could use a bit more detail, like "`REVISION_RECEIVE` should add the revision to state.items".